File: ruby/raindrops/raindrops.rb
class Raindrops



  def self.convert num

    result = ""

    if num % 3 == 0
      result += "Pling"
    end
    if num % 5 == 0
      result = result + "Plang"
    end
    if num % 7 == 0
      result = result + "Plong"
    end

    if (( num % 3 != 0) && ( num % 5 != 0) && ( num % 7 != 0))
      result = num.to_s
    end

    result
  end
end

File: javascript/raindrops/raindrops.js
function Raindrops() {


  this.convert = function(num) {
    result = "";

    if (num % 3 === 0) {
      result = result + "Pling";
    }
    if (num % 5 === 0) {
      result = result + "Plang";
    }
    if (num % 7 === 0) {
      result = result + "Plong";
    }

    if (( num % 3 !== 0 ) && ( num % 5 !== 0) && (num %7 !== 0)) {
      result = num.toString();
    }

    return result;
  };
}

module.exports = Raindrops;
